src/fsharp/LanguageFeatures.fs

```diff
@@ -28,6 +28,7 @@ type LanguageFeature =
     | NameOf = 6
     | ImplicitYield = 7
     | OpenStaticClasses = 8
+    | DotlessFloat32 = 9
```

Comment:
`DotlessFloat32Literal` would be a bit more descriptive.

Reply:
Done.